public/static/js/distros.js

```diff
@@ -280,6 +308,14 @@ mciModule.controller('DistrosCtrl', function($scope, $window, $location, $anchor
   }
 
   $scope.saveConfiguration = function() {
+    // Convert from UI display units (sec) to nanoseconds
+    if($scope.activeDistro.planner_settings.target_time > 0) {
```

Comment:
It may be cleaner to always use seconds in the client, and convert on the server since you have duration types to work with (but up to you)